src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -0,0 +1,296 @@
+package picard.util;
+
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015 The Broad Institute
```

Comment:
year 2017

Reply:
ok, though you guys should really get some kind of build step or tool to do these across the project...